### ISSUE TITLE
Warning note to check throttle when taking off in auto/guided 

### ISSUE THREAD

**Issue by oseiler2** [DroidPlanner/Tower#1703]
I've got a feature request for a warning message when taking off in auto/guided to ensure that the throttle on the RC is checked or set to 50% before taking off. The user should be allowed to enable or disable this warning.
Background: I recently flew a guided mission, which I aborted by switching back into Stabilise on my remote. I had taken off with throttle at min and when switching to Stabilise the motors stopped (no real surprise here). Although I raised the throttle immediately my quadcopter crashed. You might argue that that's a typical beginner's fault, but I reckon it has happened and will happen to a number of other people and can be prevented in most cases by displaying a warning message. At the very least it would be a deliberate - and not accidential or unknowing - decision by the pilot to launch with zero throttle.
You could consider that the flight controller should not simply stop the motors when switching from guided to stabilise (or other non-auto modes), but I can imagine that there are some people wanting exactly that behaviour for kill switch or similar functionality. I think a pop-up warning in Tower is the least intrusive way while still preventing this issue and potential crash in most scenarios.

**Comment by chris669** [comment]
I agree with this, after a recent crash passing from Follow Me to Stabilize mode without checking my Throttle stick...
